data/moves.js

```diff
@@ -1622,7 +1622,7 @@ let BattleMovedex = {
 		basePower: 80,
 		category: "Physical",
 		desc: "Damage is calculated using the user's Defense stat, including stat stage changes. The user's Ability is used as normal.",
```

Comment:
```suggestion
		desc: "Damage is calculated using the user's Defense stat instead of Attack, including stat stage changes. The user's Ability is used as normal.",
```